packages/manager/apps/dedicated/client/app/account/user/components/newAccountForm/new-account-form-controller.js

```diff
@@ -68,14 +66,7 @@ export default class NewAccountFormController {
     // Indian subsidiary flag
     this.isIndianSubsidiary = this.user.ovhSubsidiary === IN_SUBSIDIARY;
     this.determineIsEditionDisabledByKyc(this.kycStatus);
-    this.newSupportTicketUrl = this.coreURLBuilder.buildURL(
-      'dedicated',
-      '#/support/tickets/new',
-      {
-        categoryName: NEW_SUPPORT_TICKET_PARAMS.categoryName,
-        preFetchData: NEW_SUPPORT_TICKET_PARAMS.preFetchData,
-      },
-    );
+    this.newSupportTicketUrl = SUPPORT_URLS.createTicket;
```

Comment:
Won't this result in the US customer redirected to HelpCenter, when they should stay on the manager ?

Reply:
Nope as this link is only used for IN customers